resources/templates/arm7/devcontainer.json

```diff
@@ -1,6 +1,5 @@
 {
-  "image": "armv7",
-  "name": "armv7_container",
+  "name": "Azure IoT Device C SDK for armv7 Linux",
```

Comment:
Any reason now we need remove image name?

Reply:
VS Code update the `devcontainer.json` rule. In our scenario, we provide `dockerfile` and `devcontainer.json`. We only need to provide the `name` and `dockerfile` key for `remote-containers` extension to build a container for us. No need for `image` now.
See [devcontainer reference](https://code.visualstudio.com/docs/remote/containers#_devcontainerjson-reference).
`image` is only required when using an image instead of dockerfile.